src/web_client.h

```diff
@@ -18,73 +18,76 @@ extern int web_enable_gzip, web_gzip_level, web_gzip_strategy, web_donotrack_com
-    unsigned long long id;
+    unsigned long long id;              ///< web client id
 
-    uint8_t obsolete:1;                 // if set to 1, the listener will remove this client
-                                        // after setting this to 1, you should not touch
-                                        // this web_client
+    uint8_t obsolete:1;                 ///< if set to 1, the listener will remove this client
+                                        ///< after setting this to 1, you should not touch
+                                        ///< this web_client
 
-    uint8_t dead:1;                     // if set to 1, this client is dead
+    uint8_t dead:1;                     ///< if set to 1, this client is dead
 
-    uint8_t keepalive:1;                // if set to 1, the web client will be re-used
+    uint8_t keepalive:1;                ///< if set to 1, the web client will be re-used
 
-    uint8_t mode:3;                     // the operational mode of the client
+    uint8_t mode:3;                     ///< the operational mode of the client
 
-    uint8_t wait_receive:1;             // 1 = we are waiting more input data
-    uint8_t wait_send:1;                // 1 = we have data to send to the client
+    uint8_t wait_receive:1;             ///< 1 = we are waiting more input data
+    uint8_t wait_send:1;                ///< 1 = we have data to send to the client
 
-    uint8_t donottrack:1;               // 1 = we should not set cookies on this client
-    uint8_t tracking_required:1;        // 1 = if the request requires cookies
+    uint8_t donottrack:1;               ///< 1 = we should not set cookies on this client
+    uint8_t tracking_required:1;        ///< 1 = if the request requires cookies
 
-    int tcp_cork;                       // 1 = we have a cork on the socket
+    int tcp_cork;                       ///< 1 = we have a cork on the socket
 
-    int ifd;
-    int ofd;
+    int ifd; ///< ktsaou: Your help needed
+    int ofd; ///< ktsaou: Your help needed
 
-    char client_ip[NI_MAXHOST+1];
-    char client_port[NI_MAXSERV+1];
+    char client_ip[NI_MAXHOST+1];   ///< IP address of the client
+    char client_port[NI_MAXSERV+1]; ///< Port of the client
 
-    char decoded_url[URL_MAX + 1];  // we decode the URL in this buffer
-    char last_url[URL_MAX+1];       // we keep a copy of the decoded URL here
+    char decoded_url[URL_MAX + 1];  ///< we decode the URL in this buffer
+    char last_url[URL_MAX+1];       ///< we keep a copy of the decoded URL here
 
-    struct timeval tv_in, tv_ready;
+    struct timeval tv_in;    ///< ktsaou: Your help needed
+    struct timeval tv_ready; ///< timestamp client became ready
 
-    char cookie1[COOKIE_MAX+1];
-    char cookie2[COOKIE_MAX+1];
-    char origin[ORIGIN_MAX+1];
+    char cookie1[COOKIE_MAX+1]; ///< ktsaou: Your help needed
+    char cookie2[COOKIE_MAX+1]; ///< ktsaou: Your help needed
+    char origin[ORIGIN_MAX+1];  ///< ktsaou: Your help needed
```

Comment:
CORS.
This is used to store the `origin` HTTP header, so that netdata enables CORS for this specific origin instead of `*`.